docs/developer-guide/get-started/emt-build-and-deploy.md

```diff
@@ -82,17 +86,101 @@ skip to [Create a bootable USB installer](#2-create-a-bootable-usb-installer).
 
 7. Build the toolkit image:
 
-   All available image configurations are stored in JSON files, located in the `imageconfigs` folder. If you want to configure any additional kernel command line arguments, follow the instructions in [Architecture Overview](../emt-architecture-overview.md#kernel-command-line).
-   For example, to build a RAW image without real-time extensions, use `edge-image.json` and run
-   the following command:
+   > **IMPORTANT**: Add the `REBUILD_PACKAGES=n` parameter only for the latest tags,
+   > as the build process uses prebuilt RPMs - downloads the latest available packages
+   > from the RPM repository.
 
    ```bash
    sudo make image -j8 REBUILD_TOOLS=y REBUILD_PACKAGES=n CONFIG_FILE=./imageconfigs/edge-image.json
    ```
 
-   The built image will be located in its separate directory under  `edge-microvisor-toolkit/out/images/`.
+### 1B. Build from a release tag
+
+1. Clone the release branch of edge-microvisor-toolkit repository:
+
+   Check the [tags](https://github.com/open-edge-platform/edge-microvisor-toolkit/tags) for
+   the `<release_tag_name>`. For example,
+   [3.0.20250718](https://github.com/open-edge-platform/edge-microvisor-toolkit/releases/tag/3.0.20250718).
+
+   ```bash
+   git clone https://github.com/open-edge-platform/edge-microvisor-toolkit --branch=3.0.20250718
+   ```
+
+2. Navigate to the `edge-microvisor-toolkit` directory:
+
+   ```bash
+   cd edge-microvisor-toolkit
+   ```
+
+3. Install prerequisites:
+
+   > **Note:** Requirements for building the toolchain were validated on Ubuntu 22.04 host.
+
+   ```bash
+   # Install required dependencies.
+   sudo ./toolkit/docs/building/prerequisites-ubuntu.sh
+
+   # Also supported:
+   #    make -C toolkit install-prereqs
+
+   # Fix go 1.21 link
+   sudo ln -vsf /usr/lib/go-1.21/bin/go /usr/bin/go
+   sudo ln -vsf /usr/lib/go-1.21/bin/gofmt /usr/bin/gofmt
```

Comment:
this should be 1.23 
https://github.com/open-edge-platform/edge-microvisor-toolkit/blob/3.0/toolkit/docs/building/prerequisites-ubuntu.sh#L62